os_stub/spdm_device_secret_lib_sample/lib.c

```diff
@@ -2015,6 +2015,11 @@ bool libspdm_psk_master_secret_hkdf_expand(
 #endif /* LIBSPDM_ENABLE_CAPABILITY_PSK_EX_CAP */
 
 #if LIBSPDM_ENABLE_CAPABILITY_SET_CERT_CAP
+bool libspdm_is_in_trusted_environment()
+{
+    return false;
```

Comment:
Should make this settable from `unit_test` so that the `true` path can be tested as well.

Reply:
Agree. Code is updated and unit tests are added.